docs/configuration/secrets/troubleshoot.md

```diff
@@ -0,0 +1,35 @@
+# Troubleshooting
+
+The sections below include information to help you troubleshoot issues that you may come across while configuring the secrets used by Codacy.
+
+If the information provided on this page is not enough to solve your issue, contact [support@codacy.com](mailto:support@codacy.com) providing all the information that you were able to collect while following the troubleshooting instructions.
+
+## Lost/Changed initially generated secrets
+
+While trying to open Codacy,
+you were shown an error message stating that,
+the secret on the database and the one in your configuration file are different.
+
+### Steps
+
+1. Get the latest logs from the API service
+
+    a. Get logs and filter by the prefix of the message
```

Comment:
I would remove this detail (since the script does a great job in abstracting this) and unindent the lines below one level.